nemo/collections/asr/modules/audio_preprocessing.py

```diff
@@ -227,6 +254,139 @@ def filter_banks(self):
         return self.featurizer.filter_banks
 
 
+class AudioToMFCCPreprocessor(AudioPreprocessor):
```

Comment:
should this be decorated with @experimental as it does not have any tests for it?

Reply:
Yes, good point

Reply:
Done